spec/requests/api/v1/npq/previous_funding/show_spec.rb

```diff
@@ -0,0 +1,113 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+RSpec.describe "NPQ Funding API", type: :request do
```

Comment:
Do we want to add a test when all params are sent?

Reply:
Good call